src/lib/y2storage/boot_requirements_strategies/analyzer.rb

```diff
@@ -570,7 +570,10 @@ def in_bcache?(device)
           # If this is not a BlkFilesystem (e.g. NFS), it can't be in a BCache
           return false unless device.respond_to?(:plain_blk_devices)
 
-          device.plain_blk_devices.any? { |dev| dev.is?(:bcache) }
+          # Let's trust in the hierarchy to determine if the device belongs to a Bcache, although
+          # there may be some advanced configuration for which this is not totally true because the
+          # device is reachable from a not BCache ancestor.
```

Comment:
Not sure if the comment is clear enough. What about this?

Strictly speaking, with very advanced storage configurations it may be possible to access a filesystem
with bcache ancestors in the devicegraph without actually accessing the bcache. But that would be an
extreme case and is not supported by YaST.